.github/pull_request_template.md

```diff
@@ -5,11 +5,19 @@ Things the PR does go here
 - points like this
 - if you like
 
+### Pre-PR Checklist:
+
+Have you
+- [ ] Updated NUbook if necessary
+- [ ] Added/Updated tests for your changes, including regression tests for bug fixes
+- [ ] Updated relevant module READMEs
+
 ## Optional Headings
```

Comment:
Should this be a heading 3?

Reply:
It's a higher level than the others that follow it, because they're all _optional_, so I deliberately made it a bigger heading.

Reply:
Gotcha sorry I thought that was an example of how to add a custom optional heading

Reply:
Having a level 2 heading might be an accessibility issue. I feel like GitHub should handle this itself, but PR comments live under a level 2 heading. It might be confusing to have this level 2 here.

Reply:
I want to communicate to people that they should remove the headings they don't use, and they should also remove that heading `## Optional Headings` in all cases. I am not sure how to go about that though. Do you have any suggestions?

Reply:
I'd suggest increasing the heading level of all of them by one. Maybe add in a line under this heading saying something along the lines of 'delete anything in this section that is not needed (including this line)'.

Reply:
Oh wait but they're removing the heading. Maybe use a line break.